extensions/amp-subscriptions/0.1/entitlement.js

```diff
@@ -119,7 +128,18 @@ export class Entitlement {
     const products = json['products'] || [];
     const subscriptionToken = json['subscriptionToken'];
     const loggedIn = json['loggedIn'];
-    return new Entitlement(source, raw, /* service */ '',
-        products, subscriptionToken, loggedIn);
+    const meteringData = json['metering'];
+    let metering = null;
+    if (meteringData) {
+      metering = {
+        quotaLeft: meteringData['quotaLeft'],
+        quotaTotal: meteringData['quotaTotal'],
+        durationLeft: meteringData['durationLeft'],
```

Comment:
This should be simply a `resetTime` with a time_t value.

Reply:
Done